package.json

```diff
@@ -10,7 +10,7 @@
     "url": "https://github.com/ministryofjustice/prepare-a-case.git"
   },
   "scripts": {
-    "start": "DISPLAY= docker compose up",
+    "start": "IP=$(ipconfig getifaddr en0) && /usr/X11/bin/xhost + $IP && DISPLAY=$IP:0 docker compose up",
```

Comment:
Another that I'm unsure will work on non Mac. It isn't used on the CI though. Perhaps a start-mac or some type of script for setting it before it runs docker compose up?